modules/builds-create-imagestreamtag.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//* assembly/builds
+//* builds/running-entitled-builds.adoc
+
+[id="builds-create-imagestreamtag-{context}"]
+= Creating an imagestreamtag to the Red Hat Container Catalog
+
+To run entitled builds, you must create an `imagestreamtag` to
+`registry.access.redhat.io/rhel7/rhel:latest`.
+
+.Procedure
+----
+$ curl -k \
```

Comment:
Two approaches - now that UBI has shipped:
1) into a single project: `oc tag --source=docker registry.redhat.io/ubi7/ubi:latest myproject/ubi:latest`
2) into the `openshift` namespace, making it available to developers in all projects: `oc tag --source=docker registry.redhat.io/ubi7/ubi:latest openshift/ubi:latest`

Reply:
(note this example requires a pull secret since it's using the TBR)

Reply:
Yes, need to create a pull secret for registry.redhat.io, and link it to user project. I think it's better tree with: https://github.com/openshift/openshift-docs/pull/13988/files#r263808990, what do you think @bmcelvee ? , it's seems more clear.

Reply:
@adambkaplan and @wewang58 - thanks! 

I tried to keep it to Adam's org tree. Do any of the titles need changing?

Reply:
@bmcelvee  I think you can reference the doc: http://pastebin.test.redhat.com/761959, @adambkaplan what do you think? thanks.